Change type inferred from empty annotation

ansley opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#45360 Change type inferred from empty annotation**

Differential Revision: [D24078645](https://our.internmc.facebook.com/intern/diff/D24078645)

Commit:
Change type inferred from empty annotation

[ghstack-poisoned]

Commit:
Update on "Change type inferred from empty annotation"




[ghstack-poisoned]

gmagogsfm reviewed (commented):
Could you add a test? Thanks.

Commit:
Update on "Change type inferred from empty annotation"




[ghstack-poisoned]

Commit:
Update on "Change type inferred from empty annotation"




[ghstack-poisoned]

Commit:
Update on "Change type inferred from empty annotation"


Differential Revision: [D24078645](https://our.internmc.facebook.com/intern/diff/D24078645)

[ghstack-poisoned]

Commit:
Update on "Change type inferred from empty annotation"


Differential Revision: [D24078645](https://our.internmc.facebook.com/intern/diff/D24078645)

[ghstack-poisoned]
